### PR TITLE
Add plugin dependencies documentation

### PR DESCRIPTION
**Pull request by ilyin371**

### CODE REVIEW

**Comment by ilyin371** [comment]
[Document how to use new plugin dependency for graph apps](https://trello.com/c/2eFtzN4B/984-document-how-to-use-new-plugin-dependency-for-graph-apps)

**Comment by Metaur** [comment]
@oskarhane should we wait to merge this before the release?

**Comment by oskarhane** [comment]
Yes, let's wait @Metaur

File: README.md [hunk 1]
```diff
@@ -332,6 +332,28 @@ If the permission has not been already granted, the system dialog box is shown:
 
 ![Permission Request](images/requestPermission.png)
 
+
+## Plugin dependencies
+
+A graph application may specify database plugins required for the graph app to work. It is applicable for any locally installed and on-premise graph apps.
+To specify the dependencies, add `pluginDependencies` section to `manifest.json` file.
```
Comment (oskarhane) [inline README.md]: `package.json`

Comment (ilyin371) [reply README.md]: It's actually in `manifest.json`.

**Pull request merged**